tests/test_core.py

```diff
@@ -17,15 +17,15 @@
         ("core__condition"),
         ("core__documentreference"),
         ("core__encounter"),
-        ("core__encounter_type"),
         ("core__medication"),
         ("core__medicationrequest"),
         ("core__observation"),
         ("core__observation_lab"),
+        ("core__patient"),
```

Comment:
this has been missing for awhile! 😬

Reply:
Do you know the reason that `core__patient` has been removed? 
**FHIR Patient object** exists. I think the reason was because of patient AGE or DOB, which we just truncated down to YYYY. Otherwise, I think this table "should" exist?

Reply:
DPH active Cumulus studies have all been "over time" and thus relied more on `core__encounter`, but I can easily imagine a user (not me) being confused by the lack of a `core__patient` table.

Reply:
This was removed, or not added, only from our unit tests, as an oversight. I have not done the work to trace this back